features/user_login.feature

```diff
@@ -11,12 +11,12 @@ Scenario: User Login
   And I fill in "Email" with "bob@example.com"
   And I fill in "Password" with "abcde123"
   And I click "Log in" button
-  Then I should see the message "Signed in successfully."
+  Then I should see "Signed in successfully."
 
 Scenario: User Login with invalid password
   When I visit "landing" page
   And I click "Login" button
   And I fill in "Email" with "bob@bob.com"
   And I fill in "Password" with "bob1234S"
   And I click "Log in" button
-  Then I should see the message "Invalid Email or password."
+  Then I should see "Invalid Email or password."
```

Comment:
I actually preferred `Then I should see the message "Signed out successfully."`. Or perhaps refactor it to `Then I should see a flash message "Signed out successfully."` and make sure it is actually a css class (in your step definition)?